htdocs/compta/cashcontrol/report.php

```diff
@@ -135,6 +135,8 @@
 	dol_print_error('Value for key = '.$key.' not supported');
 	exit;
 }*/
+if ($object->date_valid==null) $object->date_valid=dol_now();
+$sql .= " AND f.date_closing BETWEEN '".$db->idate($object->date_creation)."' AND '".$db->idate($object->date_valid)."'";
```

Comment:
Don't understand this condition.
Process is creation -> validation -> closing.
What is goal of testing if invoice was closed before being validated ?